packages/api/test/codegen/languages/typescript/index.test.ts

```diff
@@ -200,37 +179,6 @@ describe('typescript', () => {
             expect(res.constructor.name).toBe('Response');
           });
       });
-
-      /**
-       * This test is impossible to run now because its a `.js` file that's loading ESM code. The
-       * CJS SDK we're generating here should have a `.cjs` extension but we're going to overhaul
-       * this entire codegen process with `tsup` so this test is being skipped for now.
-       *
-       * @see {@link https://github.com/readmeio/api/pull/734}
-       */
-      // it.skip('should be able to make an API request (JS + CommonJS)', async () => {
-      //   const sdk = await import('../../../__fixtures__/sdk/simple-js-cjs/index.js').then(r => r.default);
-      //   fetchMock.get('http://petstore.swagger.io/v2/pet/findByStatus?status=available', mockResponse.searchParams);
-
-      //   await sdk.findPetsByStatus({ status: ['available'] }).then(({ data, status, headers, res }) => {
-      //     expect(data).toBe('/v2/pet/findByStatus?status=available');
-      //     expect(status).toBe(200);
-      //     expect(headers.constructor.name).toBe('Headers');
-      //     expect(res.constructor.name).toBe('Response');
-      //   });
-      // });
-
-      it('should be able to make an API request (JS + ESM)', async () => {
```

Comment:
what if we added a test in this file that imports the generated `cjs` file and ensures that that works?

Reply:
thats going to require building a dist for these fixtures and keeping that dist in git. i want to move these fixtures into `@api/test-utils` at some point, will rework this then